docs/JMAQS_1/appium/AppiumFAQ.md

```diff
@@ -0,0 +1,30 @@
+# <img src="resources/jmaqslogo.jpg" height="32" width="32"> Appium FAQ
+
+## What Browsers can I use?
```

Comment:
Looks like this is all from the selenium FAQ

Reply:
Left this page blank and added it to Update Documentation Issue #283